src/sources/typeorm/resources/person.resource.ts

```diff
@@ -0,0 +1,41 @@
+import {CreateResourceResult} from "../../../admin/create-resource-result.type";
+import {menu} from "../../../admin";
+import {Person} from "../models";
+import {DETAILED_STATS, DONT_TOUCH_THIS_ACTION} from "../../../admin/components.bundler";
+import {validateEmail} from "../handlers/validate-email.handler";
+
+export const CreatePersonResource = (): CreateResourceResult<typeof Person> => ({
+  resource: Person,
+  options: {
+    parent: menu.typeorm,
+    actions: {
+      new: {
+        before: [validateEmail]
+      },
+      edit: {
+        before: [validateEmail]
+      },
+      detailedStats: {
+        actionType: 'resource',
+        icon: 'Apps',
+        name: 'Resource statistics',
+        component: DETAILED_STATS,
+        handler: async (request, response, data) => {
+          return {true: 'ueas'}
+        },
+        showInDrawer: true,
+      },
+      dontTouchThis: {
+        actionType: 'record',
+        icon: 'Exit',
+        guard: 'You can setup guards before an action - just in case.',
```

Comment:
I think this `You can setup guards before an action - just in case.` goes to i18n so it will be uppercased?

Reply:
![image](https://user-images.githubusercontent.com/60508566/155132478-6d47a6c0-5bc2-442f-9577-7d7c556909c8.png)
Yup, any ideas?

Reply:
`guard: 'youCanSetupGuards'`
+ add `youCanSetupGuards` to locale (`messages`, not sure if it needs to go to specific resource or common)